package.json

```diff
@@ -33,8 +33,11 @@
     "@angular/platform-browser": "~10.2.1",
     "@angular/platform-browser-dynamic": "~10.2.1",
     "@angular/router": "~10.2.1",
+    "echarts": "^4.7.0",
+    "echarts-gl": "^1.1.1",
     "echarts": "^5.0.0",
     "echarts-gl": "^2.0.0-rc.3",
+        "@types/echarts": "^4.9.2",
```

Comment:
possible to format this file?

Reply:
This package or package.json?

Reply:
the indentation just doesn't look right, I think the file need to be formatted

Reply:
Got it. I'll fix it now